app/src/main/java/org/wikipedia/page/PageActivity.kt

```diff
@@ -666,20 +666,16 @@ class PageActivity : BaseActivity(), PageFragment.Callback, LinkPreviewDialog.Ca
         if (!Prefs.showOneTimeCustomizeToolbarTooltip) {
             return
         }
-        binding.pageToolbarButtonShowOverflowMenu.postDelayed({
-            if (!isDestroyed) {
-                themeTooltipBalloon =
-                    FeedbackUtil.getTooltip(this, getString(R.string.theme_chooser_menu_item_short_tooltip), arrowAnchorPadding = -DimenUtil.roundedDpToPx(12f), topOrBottomMargin = -12, aboveOrBelow = true, autoDismiss = false, showDismissButton = true, callback = object :
-                        FeedbackUtil.Callback {
-                        override fun onDismissClicked() {
-                            Prefs.showOneTimeCustomizeToolbarTooltip = false
-                            Prefs.toolbarTooltipVisible = false
-                        }
-                    })
-                themeTooltipBalloon?.showAlignBottom(binding.pageToolbarButtonShowOverflowMenu)
-                Prefs.toolbarTooltipVisible = true
+        if (!isDestroyed) {
```

Comment:
Since the logic is not inside the `postDelayed`, can we remove the logic of checking `isDestroyed`?